src/RazorSdk/Tool/Microsoft.NET.Sdk.Razor.Tool.csproj

```diff
@@ -19,6 +19,15 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Razor.Language" Version="$(MicrosoftAspNetCoreRazorLanguageVersion)" />
     <PackageReference Include="Microsoft.CodeAnalysis.Razor" Version="$(MicrosoftCodeAnalysisRazorVersion)" />
+    <!-- We include a package reference to the M.C.C version used in the repo to ensure that NuGet's dependency
+    resolution favors this version when writing the assembly reference metadata to `rzc.dll`. Without this package
+    reference, NuGet will resolve the version of M.C.C at the next level of the dependency tree. M.A.R.L uses M.C.C
+    version 3.8.0. By including this reference, NuGet will resolve the assembly associated with $(MicrosoftCodeAnalysisCSharpPackageVersion)
+    since it is higher up the dependency tree.
+
+    This is needed to ensure that the assembly reference to M.C.C in rzc.dll always matches what is available in
+    `Roslyn/bincore` for our consolidate loading logic. -->
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpPackageVersion)" />
```

Comment:
This seems to be causing failure in installer https://github.com/dotnet/installer/pull/10625